lstchain/irf/hdu_table.py

```diff
@@ -8,15 +8,20 @@
 from astropy.time import Time
 
 #from lstchain.reco.utils import get_effective_time
+from lstchain.version import get_version
 
 __all__ = ["create_obs_hdu_index", "create_event_list"]
 
 log = logging.getLogger(__name__)
 
 DEFAULT_HEADER = fits.Header()
+DEFAULT_HEADER["CREATOR"] = f"lstchain v{get_version(pep440=True)}"
 DEFAULT_HEADER["HDUDOC"] = "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats"
 DEFAULT_HEADER["HDUVERS"] = "0.2"
 DEFAULT_HEADER["HDUCLASS"] = "GADF"
+DEFAULT_HEADER["ORIGIN"] = "CTAO"
```

Comment:
CTAO? we are not there yet, right?

Reply:
I was confused about whether to keep it as CTA or CTAO. But, yes, it is early to have it as CTAO